gen3-integration-tests/gen3_ci/jenkins_jobs/check-kube-pod.groovy

```diff
@@ -0,0 +1,97 @@
+/*
+    String parameter JOBNAME
+    eg - manifest-indexing
+
+    String parameter LABELNAME
+    eg - sowerjob
+
+    String parameter EXPECTFAILURE
+    default value - False
+*/
+pipeline {
+    agent {
+        node {
+            label 'gen3-ci-worker'
+        }
+    }
+    stages {
+        stage('Clean old workspace') {
+            steps {
+                cleanWs()
+            }
+        }
+        stage('Initial Setup') {
+            steps {
+                // cloud-automation
+                checkout([
+                  $class: 'GitSCM',
+                  branches: [[name: 'refs/heads/master']],
+                  doGenerateSubmoduleConfigurations: false,
+                  extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'cloud-automation']],
+                  submoduleCfg: [],
+                  userRemoteConfigs: [[credentialsId: 'PlanXCyborgUserJenkins', url: 'https://github.com/uc-cdis/cloud-automation.git']]
+                ])
+            }
+        }
+        stage('Check Kube Pod') {
+            options {
+                    timeout(time: 5, unit: 'MINUTES')   // timeout on this stage
+                }
+            steps {
+                dir("check-kube-pod"){
+                    script {
+                        try {
+                            sh '''#!/bin/bash +x
+                            set -e
+                            export GEN3_HOME=\$WORKSPACE/cloud-automation
+                            export KUBECTL_NAMESPACE=\${NAMESPACE}
+                            source $GEN3_HOME/gen3/gen3setup.sh
+
+                            N_ATTEMPTS=10
+                            echo "NAMESPACE : $KUBECTL_NAMESPACE"
+                            for (( i=1; i<=N_ATTEMPTS; i++ )); do
+                                sleep 10
```

Comment:
the `N_ATTEMPTS` logic may be unnecessary since there is already a 5 min timeout in the stage options - just a suggestion, but maybe we can simplify by removing one of the 2. We could just use a `while true` loop and let the timeout decide when to give up